index.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Tochukwu Orizu</title>
+    <link rel="stylesheet" href="/styles/style.css">
+
+</head>
+<body>
+    <header class="header">
+        <div class="logo">My logo</div>
+        <nav class="navigation">
+            <img src="/img/Icons/Union.png">
+        </nav>
+    </header>
+
+    <section class="container">
+        <h1 class="box1">I'm Tochukwu Glad to see you!</h1>
+        <p class="box2">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        <p class="box3">LET'S CONNECT</p>
+       
+    </section>
+    <footer class="socials">
+        <img src="/img/social media icons.png">
+    </footer>
+</body>
+</html>
```

Comment:
kindly you need to fix your margin and your padding. there is too much space between your footer and the end of the page. please make sure you're following he [Figma](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/images/m1_setup/headline-mobile.png) design. inspect the page to know the padding and the margin of your template in Figma. often it is symbolized by a `*`